src/workloads/query/TimeseriesTSBSQuery.yml

```diff
@@ -0,0 +1,435 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This workload tests running identical queries as tsbs_query in a Genny workload. During
```

Comment:
I'd change the description to say:
"This workload runs queries from the tsbs_query benchmark in Genny. It allows us to cross-check the results of the TSBS benchmarks, collect additional measurements, profile individual queries, and extend some of the queries as needed.
The data is preloaded by the dsi configuration. See 'configurations/test_control/test_control.tsbs_query.yml' for all the details. There are 20736000 documents in the collection."

Reply:
Also, think about whether there are any queries we'd like to run _in addition_ to the ones from tsbs_query against this dataset. E.g. non-trivial match on a measurement?

Reply:
Right now I'm more worried about having workloads for all the optimizations we do on time-series. Benchmarking typical user queries would be nice but not as pressing, imho.